.github/workflows/black_python_formatter.yml

```diff
@@ -0,0 +1,42 @@
+name: Black Python Formatter
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    # Name the Job
+    name: Formatter
+    # Set the agent to run on
+    runs-on: ubuntu-latest
```

Comment:
This should be pinned at a specific version so it doesn't break without warning on us at some undefined point in the future when the definition of what constitutes `latest` changes.